Haxe/Externs/unreal/FOutputDeviceConsole.hx

```diff
@@ -0,0 +1,34 @@
+package unreal;
+
+import unreal.FOutputDevice;
+
+@:glueCppIncludes("Misc/OutputDeviceConsole.h")
+@:noCopy
+@:noEquals
+@:uextern extern class FOutputDeviceConsole extends FOutputDevice {
+  /**
+    Global console logger singleton
+    Note: FWindowsPlatformOutputDevices::GetLogConsole() will make a new instance, so don't call it.
+   **/
+  @:global static var GLogConsole(default, never) : PExternal<FOutputDeviceConsole>;

```

Comment:
This is just a small nitpick: Sometimes @:global variables aren't declared on the same header as the root `glueCppIncludes` (The `GLog` case is an exception - since it is defined there). So it's a good practice to add a `@:glueCppIncludes` for the global variable as well. On the GLogConsole case, it's at `CoreGlobals.h`